infrastructure/charts/agent/templates/apisixroute.yaml

```diff
@@ -136,6 +136,37 @@ spec:
 
 ---
 
+apiVersion: apisix.apache.org/v2
+kind: ApisixRoute
+metadata:
+  name: agent-credential-definition-registry-route
+  namespace: "{{ .Release.Namespace }}"
+  labels:
+    {{ template "labels.common" . }}
+spec:
+  http:
+  - name: agent-credential-definition-registry-rule
+    match:
+      hosts:
+      {{- range .Values.ingress.applicationUrls }}
+      - {{ . }}
+      {{- end }}
+      paths:
+        - /prism-agent/credential-definition-registry/definitions/*
+      methods:
+        - GET
+    backends:
+      - serviceName: agent-server-tapir-service
+        servicePort: 8085
+    plugins:
+    - name: proxy-rewrite
+      enable: true
+      config:
+        regex_uri: ["^/prism-agent/credential-definition-registry/definitions/(.*)", "credential-definition-registry/definitions/$1"]
```

Comment:
Do we need this alias?
It will introduce the different routes defined in the OAS and different routes in the Agent when running locally (or without APISIX API gateway)

Reply:
I propose to keep this as a side task (tech debt), to figure out what is the best format and then convert all diagrams in a single step.

Reply:
Good point, Yurii. It seems this one is a duplicate of the previous one. I'll remove it 👍